Apps/APN/src/components/BuildInfoText.js

```diff
@@ -12,7 +12,7 @@ const BuildInfoText = () => {
     const value =
       `Customer.io` +
       ` React Native SDK ${sdkPackageJson.version}` +
-      ` Ami App ${getVersion()} (${getBuildNumber()})`;
+      ` APN Sample ${getVersion()} (${getBuildNumber()})`;
```

Comment:
Shouldn't this be `RN APN Sample .....` ?

Reply:
The string was getting too lengthy so I deliberately dropped RN prefix as it was slightly repetitive 🤓 

The final string looks like

```
Customer.io React Native SDK 3.1.3 APN Sample 1.0 (1)
```